inference-engine/tests/functional/plugin/cpu/subgraph_tests/src/move_eltwise_up_data_movement.cpp

```diff
@@ -0,0 +1,224 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include <ngraph/pass/visualize_tree.hpp>
+#include <ngraph/function.hpp>
+#include <ngraph/opsets/opset8.hpp>
+#include <transformations/init_node_info.hpp>
+#include <ngraph/pass/manager.hpp>
+
+#include "common_test_utils/ngraph_test_utils.hpp"
+#include "ngraph_transformations/move_eltwise_up_data_movement.hpp"
+
+using namespace testing;
+using namespace ngraph;
+
+namespace SubgraphTestsDefinitions {
+
+class MoveEltwiseUpThroughDataMovFixture : public ::testing::TestWithParam<int64_t> {
+};
+
```

Comment:
These tests do not run or use CPU plugin. They just test the transformation. So why are they placed in the cpu subgraph test folder. However, we still need tests that will check that this transformation actually works in the plugin, and the plugin works correctly with this transformation activated.